src/main.c

```diff
@@ -18,15 +19,9 @@
 #include "mixer.h"
 #include "rc.h"
 
-extern void SetSysClock(bool overclock);
-
-serialPort_t *Serial1;
-
 int main(void)
```

Comment:
Haha, yeah. I was thinking we could maybe just have like a `main_loop` or `run` function or something that just lumps it all together; do you think there's value in having it split up? We could call it `init()` and `run()` maybe so it doesn't sound quite so arduino-y